repro/Proxy.cxx

```diff
@@ -558,12 +558,16 @@ Proxy::addClientTransaction(const Data& transactionId, RequestContext* rc)
 }
 
 void
-Proxy::postTimerC(std::unique_ptr<TimerCMessage> tc)
+Proxy::postTimerC(std::unique_ptr<TimerCMessage> tc, int customDelayMs)
 {
-   if(mTimerC > 0)
+   // Convert ms to seconds
+   int delay = customDelayMs > 0 ? std::round(customDelayMs / 1000) : 0;
```

Comment:
FYI - You can use SipStack::postMS to post with a millisecond delay instead and not lose the precision on new method parameter units (if it must be MS for some reason).

Reply:
The problem here is that TimerB is configured in milliseconds (in `Timer.cxx`) while TimerC is configured in seconds (in `repro.config`). I see 2 options here:
1. timer values are configured in different units to avoid additional conversion code. This could lead to some user/configuration errors in case of `TxOptions` where 2 timer values appear next to each other.
2. a common unit is chosen and extra conversion is performed

I've opted for the option 2 at the moment, but happy to switch to option 1 if you think the code simplicity outweighs the risk of misconfiguration.

Reply:
Actually I'd like to see the units in the variable names regardless, I think this really helps developers from needing to dig around to see how they are used.